kornia/augmentation/container/augment.py

```diff
@@ -1,23 +1,25 @@
 import warnings
 from itertools import zip_longest
-from typing import Any, cast, List, Optional, Tuple, Union
+from typing import Any, List, Optional, Tuple, Union, cast
 
 import torch
 
-from kornia.augmentation.base import _AugmentationBase, GeometricAugmentationBase2D, IntensityAugmentationBase2D
+from kornia.augmentation._2d.geometric.base import GeometricAugmentationBase2D
+from kornia.augmentation._2d.intensity.base import IntensityAugmentationBase2D
```

Comment:
since it started with the underscore, so I would suggest to hide these packages
```suggestion
from kornia.augmentation import GeometricAugmentationBase2D
from kornia.augmentation import IntensityAugmentationBase2D
```